src/MultichannelMarketing/GLAChannel.php

```diff
@@ -198,7 +198,7 @@ function ( array $campaign_data ) use ( $currency ) {
 						(string) $campaign_data['id'],
 						$this->campaign_types['google-ads'],
 						$campaign_data['name'],
-						admin_url( 'admin.php?page=wc-admin&path=/google/settings&subpath=/campaigns/edit' ),
```

Comment:
I think this one is still missing one parameter: `&programId=19557946999`

So it should be, I think,
```php
admin_url( 'admin.php?page=wc-admin&path=/google/settings&subpath=/campaigns/edit' . $campaign_data['id'] )
```